rules/downgrade/tests/Rector/Coalesce/DowngradeNullCoalescingOperatorRector/Fixture/fixture.php.inc

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Rector\Downgrade\Tests\Rector\Coalesce\DowngradeNullCoalescingOperatorRector\Fixture;
+
+$array = [];
+$array['user_id'] ??= 'value';
```

Comment:
Could you wrap this into PHP class as other fixtures?
Now this code gets executed, as this file is included so test can use reflection and other static analysis for it

Reply:
Oops, I was not aware of the issue, fixed now